### PR TITLE
Improve some #line directives

### PR DESCRIPTION
**Pull request by mandolaerik**

### CODE REVIEW

**Comment by mandolaerik** [comment]
Part of #220

File: py/dml/c_backend.py [hunk 1]
```diff
@@ -1576,15 +1576,15 @@ def generate_initialize(device):
...
-                # Call hard_reset
-                hard_reset = codegen_call_byname(device.site, device, (),
-                                                 'hard_reset', [], [])
-
-            mkCompound(device.site, [init, hard_reset]).toc()
+            stmts = []
+            for method_name in ['init', 'hard_reset']:
+                method = device.get_component(method_name, 'method')
+                with InitFailure(method.site):
+                    # Inline the init method
```
Comment (lwaern-intel) [inline py/dml/c_backend.py:1583]: Get rid of this comment.

**Comment by lwaern-intel** [comment]
This will be redundant with #221, but can be allowed as a stop-gap since #221 is rapidly expanding its complexity.

**Comment by syssimics** [comment]
Verification [#12332186](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=12332186): pass

**Comment by mandolaerik** [comment]
[Test MCA L](https://teamcity01-ir.devtools.intel.com/viewQueued.html?itemId=12334000)

**Review by lwaern-intel — CHANGES_REQUESTED** [review]
get rid of it and this is an easy approve.

File: py/dml/c_backend.py (hunk 1 above)
Comment (mandolaerik) [reply py/dml/c_backend.py]: thanks, removed

**Comment by syssimics** [comment]
Verification [#12335683](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=12335683): pass

**Review by lwaern-intel — APPROVED** [review]

**Pull request merged**